scopes/compositions/compositions/compositions.tsx

```diff
@@ -32,11 +33,10 @@ export function Compositions() {
 
     selectComposition(next);
   }, [component]);
-
-  const [isSidebarOpen, setSidebarOpenness] = useState(component.compositions.length > 0);
+  const isMobile = useIsMobile();
+  const showSidebar = useMemo(() => !isMobile && component.compositions.length > 0, [component.compositions.length]);
```

Comment:
useMemo is unnecessary. Cheaper to just calculate than to start caching response

Reply:
fixed